### ISSUE TITLE
[BUG] unexpected persistency of data access password labels

### ISSUE THREAD

**Issue by ccacciari** [UtrechtUniversity/yoda-ansible#209]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I create a data access password in Yoda and assign a label to it, for example "test1".
After a while the password expires and is not listed anymore in my list of data access passwords.
I create a new data access password and I assign the same label "test1", then I got an error:

![image](https://user-images.githubusercontent.com/7924423/216383943-1fec395f-5c6b-4c88-8629-3f61b3449588.png)

### Expected Behavior

I would expect that the labels of expired passwords can be reused.
Or, at least, that they remain visible for the user, so that he can avoid to guess every time which label has already been used.

### Steps To Reproduce

1. create a data access password in Yoda and assign a label to it, for example "test1".
2. after a while the password expires and it is not listed anymore in the list of data access passwords.
3. then create a new data access password and assign the same label "test1"

### Environment

```markdown
- Yoda:1.8.3
- Ansible:
- Operating System:CentOS 7
- Browser:
```

**Comment by stsnel** [comment]
Thanks for reporting. We're tracking this issue as YDA-4925 in our internal bug database. It has recently been fixed in the development branch. This fix has currently not been merged into 1.8.x yet. I hope to be able to provide an estimated timeline for a new release with this fix in roughly a week.

**Comment by lwesterhof** [comment]
Fixed in https://github.com/UtrechtUniversity/yoda-ruleset/commit/835cd4fa1b7f33f1ff2a2cc5523389a5a0ef9677
